lib/Partitioner/Partitioner.cpp

```diff
@@ -394,6 +394,15 @@ void Partitioner::genBackendMap(
   }
 }
 
+const DeviceInfo &
+Partitioner::getDeviceInfoForBackend(llvm::StringRef backendName) {
+  for (DeviceInfo &devInfo : deviceInfo_) {
```

Comment:
I think I'm missing something, where is deviceInfo_ getting set?

Reply:
@gcatron deviceInfo_ is passed to Partitioner from HostManager when Partitioner is created (i.e. Partitioner constructor)